examples/flower_classifier/train.py

```diff
@@ -0,0 +1,70 @@
+import click
+
+from keras_image_classifier import MLflowInceptionV3, KerasImageClassifier, MLflow_VGG16
+
+import os
+
+
+def download_input():
+    import requests
+    url = 'http://download.tensorflow.org/example_images/flower_photos.tgz'
+    r = requests.get(url)
+    with open('flower_photos.tgz', 'wb') as f:
+        f.write(r.content)
+    import tarfile
+
+    with tarfile.open("flower_photos.tgz") as tar:
+        tar.extractall(path="./")
+
```

Comment:
Nit: Can we print the path to which the training data was saved?

Reply:
makes sense.